lua/modes.lua

```diff
@@ -113,6 +122,16 @@ M.define = function()
 	vim.cmd('hi ModesDelete guibg=' .. colors.delete)
 	vim.cmd('hi ModesInsert guibg=' .. colors.insert)
 	vim.cmd('hi ModesVisual guibg=' .. colors.visual)
```

Comment:
@mvllow this can be flagged `if_not_exists`, but that means highlight definition takes precedence over user configuration ?

Reply:
Sorry a little confused by this; the current implementation seems to work as expected. E.g. my theme overrides `Modes*` highlight groups and I'm still able to set custom colors in `modes.setup({ colors = { ... } })`

Reply:
NVM, my bad